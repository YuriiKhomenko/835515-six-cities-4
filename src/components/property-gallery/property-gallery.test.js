import React from "react";
import renderer from "react-test-renderer";
import PropertyGallery from "../property-gallery/property-gallery.jsx";

const images = [`img/apartment-01.jpg`, `img/apartment-02.jpg`, `img/studio-01.jpg`];

it(`Render Property Gallery component`, () => {
  const tree = renderer
    .create(<PropertyGallery
      images={images}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
