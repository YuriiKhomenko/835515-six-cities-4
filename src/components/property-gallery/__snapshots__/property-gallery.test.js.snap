// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Property Gallery component 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
    >
      <img
        alt="img/apartment-01.jpg"
        className="property__image"
        src="img/apartment-01.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="img/apartment-02.jpg"
        className="property__image"
        src="img/apartment-02.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
    >
      <img
        alt="img/studio-01.jpg"
        className="property__image"
        src="img/studio-01.jpg"
      />
    </div>
  </div>
</div>
`;
